README.rst

```diff
@@ -59,6 +59,14 @@ Installation of SFOpenBoson, as well as all required Python packages mentioned a
     $ python -m pip install qmlt
 
 
+Tests
```

Comment:
Excluding the tests here was a deliberate decision, just to make the README shorter and more concise. The aim was to provide more (development) focused details in the docs here (http://qmlt.readthedocs.io/en/latest/installing.html), but I just noticed that I forgot to put it in, oops.

Reply:
There was a typo in the previous line though, it said SFOpenBoson instead of QMLT :) Fixed it.

Reply:
So I understand I should remove this section? @josh146 ?